api/controllers/blocks.js

```diff
@@ -31,11 +32,22 @@ let library;
  */
 function BlocksController(scope) {
 	library = {
-		db: scope.db,
 		storage: scope.storage,
 		logic: scope.logic,
 		logger: scope.logger,
 	};
+
```

Comment:
I see that it is present in `db.blocks` yet. Is there any plan to remove it or move it to storage?
Why cannot we get it from anywhere else like storage for instance?